### ISSUE TITLE
Lighthouse accessibility failures

### ISSUE THREAD

**Issue by PVince81** [nextcloud/forms#1263]
Run Chrome's Lighthouse tool after creating a simple form with radio buttons and dropdowns:

<img width="423" alt="image" src="https://user-images.githubusercontent.com/277525/180016791-fc8a460a-9e64-4217-96f6-de55def778ff.png">

Note: the avatardiv is unrelated as it's in the header.

This was observed in forms version 2.5.1 on Nextcloud 23.0.7

**Comment by susnux** [comment]
Context: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?utm_source=lighthouse&utm_medium=devtools
